taxonomy-manager-common/src/main/java/com/digirati/taxman/common/rdf/RdfModelBuilder.java

```diff
@@ -52,6 +52,11 @@ public RdfModelBuilder addPlainLiteral(Property property, Map<String, String> va
         return this;
     }
 
+    public RdfModelBuilder addEmbeddedModel(Property property, RdfModel model) throws RdfModelException {
```

Comment:
This should return a `RdfModelBuilder<T>`

Reply:
Yep, I'll leave the other one untyped to make it conflict with your work a bit less.